app/installer.sh

```diff
@@ -212,6 +212,7 @@ EOF
 
   if echo "$FILENAME_VERSION" | grep -qs -E 'preview|0.0.0'; then
     URL_REPO="${REPO_REPO}-binaries-preview"
+    VERSION="${FILENAME_VERSION#*0.0.0-preview.v}"
```

Comment:
can we add a case for latest / specific commit here https://github.com/kumahq/kuma-website/blob/9948b52a52d83b0f4b7ce7800229b3dda25e2060/.github/workflows/ci.yml#L105 ?

Reply:
and we could get latest version (to check specific version) from https://kuma.io/latest_version